scripts/zephyr_module.py

```diff
@@ -233,7 +233,11 @@ def main():
         from west.util import WestNotFound
         try:
             manifest = Manifest.from_file()
-            projects = [p.posixpath for p in manifest.get_projects([])]
+            if hasattr(manifest, 'is_active'):
```

Comment:
This is a bit hacky; you can compare the `west.version.__version__` value against west 0.9. See _ensure_min_version() for examples of how to use `packaging.version.parse()` to make a high level version object which supports `>=` comparison.

Reply:
yes, I know.
But that won't allow for fetching west source code and install manually for testing this feature, as that version would be `0.8.99`.

This way allows for testing the `group` feature with Zephyr by installing `west` manually.
As soon as west `0.9.0` is released, then this code will be updated.

Reply:
> As soon as west `0.9.0` is released, then this code will be updated.

OK, so this is DNM for now until that happens, right? Adding the label.

Reply:
> > As soon as west `0.9.0` is released, then this code will be updated.
> 
> OK, so this is DNM for now until that happens, right? Adding the label.

I was actually thinking to have it in master for others to use, but I guess if people build `west` themselves, then they also know how to apply this patch to Zephyr.

Also, I see you are preparing the `0.9.0` release, #31241, which happened after I opened this PR, so `DNM` is fine.

Reply:
Finally got around to look at this again :)